html/webappapis/structured-clone/structured-clone.html

```diff
@@ -1,14 +0,0 @@
-<!doctype html>
-<title>structuredClone() tests</title>
-<meta charset=utf-8>
-<div id=log></div>
-<script src=/resources/testharness.js></script>
-<script src=/resources/testharnessreport.js></script>
-<script src=/html/webappapis/structured-clone/structured-clone-battery-of-tests.js></script>
-<script src=/html/webappapis/structured-clone/structured-clone-battery-of-tests-with-transferables.js></script>
-<script src=/html/webappapis/structured-clone/structured-clone-battery-of-tests-harness.js></script>
-<script>
-  runStructuredCloneBatteryOfTests({
-    structuredClone: (obj, transfer) => structuredClone(obj, {transfer});
```

Comment:
The semicolon on the end of this line previously caused the tests to always fail. See https://wpt.fyi/results/html/webappapis/structured-clone/structured-clone.html?label=experimental&label=master&aligned with details enabled.